.github/workflows/tests.yml

```
@@ -10,7 +10,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        python-version: [3.6, 3.7, 3.8]
+        python-version: ["3.7", "3.8", "3.9", "3.10"]

```

```suggestion
        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]

```
I still want to support Python 3.6

---

of course, added it back now d2fe39b1aa530378d1b0f27555d4d6b5011605ab My reasoning was that it hit [end of life ](https://devguide.python.org/devcycle/#end-of-life-branches) just before new year